applications/task-manager/main.cpp

```diff
@@ -99,6 +110,9 @@ int main(int argc, char **argv)
     icon_panel_create(cpu_icon_and_text, Icon::get("memory"));
     label_create(cpu_icon_and_text, "Processor");
 
+    window->ram_percent = label_create(window_root((Window*)window), "undefined%");
+    window->cpu_percent = label_create(window_root((Window*)window), "undefined%");
+    
     window->cpu_timer = timer_create(window->cpu_graph, 100, (TimerCallback)widget_cpu_update);
     timer_start(window->cpu_timer);
```

Comment:
You can set the window as the target of timer to avoid to use a global variable:

```cpp
window->cpu_timer = timer_create(window, 100, (TimerCallback)widget_cpu_update);
timer_start(window->cpu_timer);
```

Then in the timer callback, you can access the graph by doing window->cpu_graph without needing the global variable ;)